test/segment-loader_test.js

```diff
@@ -8,17 +8,18 @@
 
   var clock, xhr, requests, mse, env;
 
-  var playlistWithDuration = function(time) {
+  var playlistWithDuration = function(time, conf) {
     var
       result = {
-        mediaSequence: 0,
+        mediaSequence: conf && conf.mediaSequence ? conf.mediaSequence : 0,
         discontinuityStarts: [],
         segments: [],
         endList: true
       },
       count = Math.floor(time / 10),
       remainder = time % 10,
-      i;
+      i,
+      isEncrypted = conf && conf.isEncrypted;
```

Comment:
If this gets much more complicated, we could use `videojs.mergeOptions()` to handle merging defaults+settings.